Services/services.h

```diff
@@ -28,6 +28,7 @@
 #define STATUS_BYTE 1
 #define IN_DATA_BYTE 1
 #define OUT_DATA_BYTE 2
+#define OUT_DATA_BYTE2 5 // 6 worked! why though? 32 bits?
```

Comment:
I think I should remove this. I was using it to test something.

Reply:
Probably, if it's no longer used, then yes